docs/changelog-fragments.d/236.bugfix.md

```diff
@@ -0,0 +1,4 @@
+Ensure that Ansible calls do not return ANSI escapes, so we can parse them.
+[vscode-ansible#373] -- by {user}`ssbarnea`
```

Comment:
This is the wrong way of linking issues. The fragment should've just been symlinked.